examples/misc/lib/language_tour/functions.dart

```diff
@@ -1,5 +1,7 @@
 // ignore_for_file: unused_element, type_annotate_public_apis
 
+import 'package:meta/meta.dart';
```

Comment:
I wonder if they'll ever move required to the core and out of meta.

Reply:
Ah, right. I've added a sentence about imports. PTAL

Reply:
Kathy gave a thumbs up, so I'll merge. We can always revisit this as needed.